content/en/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -136,9 +136,14 @@ and set this flag to `false`. For example:
 
 ## {{% heading "whatsnext" %}}
 
-[Cron expression format](https://en.wikipedia.org/wiki/Cron)
-documents the format of CronJob `schedule` fields.
-
-For instructions on creating and working with cron jobs, and for an example of CronJob
-manifest, see [Running automated tasks with cron jobs](/docs/tasks/job/automated-tasks-with-cron-jobs).
-
+* Learn about [Pods](/docs/concepts/workloads/pods) and
+  [Jobs](/docs/concepts/workloads/controllers/jobs/), two concepts
```

Comment:
These pages return 404 Page not found:
- https://kubernetes.io/docs/concepts/workloads/controllers/jobs/
- https://deploy-preview-29730--kubernetes-io-main-staging.netlify.app/docs/concepts/workloads/controllers/jobs/

Reply:
This'd fix it
```suggestion
  [Jobs](/docs/concepts/workloads/controllers/job/), two concepts
```